module2/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <algorithm>
+#include <array>
+#include <cstdint>
+#include <utility>
+#include <vector>
```

Comment:
Do you need all includes here? You generally try include as little as possible in the header in favour of cpp includes,because you want to avoid cpp file including library it does not use.